packages/react-instantsearch-dom/src/components/CurrentRefinements.js

```diff
@@ -49,7 +49,7 @@ export const CurrentRefinements = ({
 const itemPropTypes = PropTypes.arrayOf(
   PropTypes.shape({
     label: PropTypes.string.isRequired,
-    value: PropTypes.func.isRequired,
+    value: PropTypes.func,
```

Comment:
initial render on client won't include `value`, since that's a function

Reply:
Rather than lower the validation we could assign a noop function to the metadata when we hydrate them.